app.js

```diff
@@ -1,13 +1,88 @@
 const express = require("express");
+const mongoose = require("mongoose");
+const Toy = require("./models/Toy.model.js");
+
 const PORT = 5005;
+
 require("dotenv").config();
 
 const app = express();
 
+app.use(express.json());
+
 app.get("/", (req, res) => {
   res.send("All good here!");
 });
 
+app.get("/toys", async (req, res) => {
+  try {
+    const toys = await Toy.find();
+    res.status(200).json(toys);
+  } catch (err) {
+    res.status(500).json({ message: "there's been an error retrieving toys" });
+  }
+});
+
+app.get("/toys/:id", async (req, res) => {
+  const { id } = req.params; //the solution is not like this one
```

Comment:
🤣

Reply:
In this case, the lab was asking you to add a query and get the toy by its name using the route `/toys/search` and a query. We want the Elfs in the toy factory to be able to search toys by name so they don't have to memorize the IDs.🧝